### DISCUSSION TITLE
Reasons for Command Failed

### DISCUSSION THREAD

**Discussion by AaronLiddiment** [keirf/greaseweazle#376] [General]
I have V4.1 and can get my Amiga 1200 internal to work perfectly but am having trouble with 3 PC 3.5" drives that I have.
GW generates a "NO INDEX" error when trying RPM command and "TRACK 0 NOT FOUND" when tring SEEK command and at no point does the drive motor spin up.
So my question is that I presume I have a hardware/cable issue but no idea where to look for the issue, so what exactly do the errors indicate and why would the motor not spin up.
I have tried using the same Amiga drive ribbon / power cable and using the --drive=a switch but it makes no difference.

**Comment by keirf** [comment]
With a straight ribbon cable the PC drives will be --drive=b
